Makefile

```diff
@@ -27,9 +27,11 @@ proto:
 	done
 
 test: testdeps
+	go test -i google.golang.org/grpc/...
```

Comment:
My GOPATH is under my home directory. If your guess is true, your proposal should fix the problem. But it does not.